src/test/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,206 @@
+            System.out.println();
+            FilesEditor();
+        } else if (inputFromConsole.equalsIgnoreCase("help")) {
+            help();
+            System.out.println();
+            FilesEditor();
+        } else if (inputFromConsole.equalsIgnoreCase("start")) {
+            System.out.println();
+            FilesEditor();
+        } else if (inputFromConsole.equalsIgnoreCase("exit")) {
+            System.exit(0);
+        } else {
+            tempTextToFile = inputFromConsole;
+            saveText();
+        }
+    }
+
+    private void saveText() {
+        System.out.println("Invalid command, do you wont to save text in file @Yes/@No?");
+        inputFromConsole = new Scanner(System.in).nextLine();
+        if (inputFromConsole.equalsIgnoreCase("Yes")) {
+            System.out.println("Write path file to save!");
+            String inputForPathFilesWrite = new Scanner(System.in).nextLine();
+            if (Files.exists(Paths.get(inputForPathFilesWrite))) {
+                System.out.println("File already exists  overwrite it @Yes/@No or add text to this file @Add?");
+                String input = new Scanner(System.in).nextLine();
+                if (input.equalsIgnoreCase("Yes")) {
+                    try {
+                        Files.write(Paths.get(inputForPathFilesWrite), tempTextToFile.getBytes());
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                    if (Files.exists(Paths.get(inputForPathFilesWrite))) {
+                        System.out.println("File created successfully");
+                        FilesEditor();
+                    }
+                } else if (input.equalsIgnoreCase("No")) {
+                    saveText();
+                } else if (input.equalsIgnoreCase("Add")) {
+                    try {
+                        Files.write(Paths.get(inputForPathFilesWrite), tempTextToFile.getBytes(), StandardOpenOption.APPEND);
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                    if (Files.exists(Paths.get(inputForPathFilesWrite))) {
+                        System.out.println("File recorded successfully");
+                        FilesEditor();
+                    }
+                }
+            } else {
+                try {
+                    Files.write(Paths.get(inputForPathFilesWrite), tempTextToFile.getBytes());
+                } catch (IOException e) {
+                    e.printStackTrace();
+                }
+                if (Files.exists(Paths.get(inputForPathFilesWrite))) {
+                    System.out.println("File created successfully");
+                    FilesEditor();
+                }
+            }
+        } else if (inputFromConsole.equalsIgnoreCase("No")) {
+            FilesEditor();
+        } else {
+            saveText();
+        }
+    }
+
+    private void help() {
```

Comment:
Optional: You can create map with key = command and value = description  and it will print either COMMAND_INFO.get(command) or COMMAND_INFO.values(). this way you can make this method much shorter.